rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -231,6 +245,39 @@ def test_thegraph_balances(
     )
 
 
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0xA9728D95567410555557a54EcA320e5E8bEa36a5']])
+@pytest.mark.parametrize('arbitrum_one_manager_connect_at_start', [(get_arbitrum_allthatnode(weight=ONE, owned=True),)])  # noqa: E501
+def test_thegraph_balances_arbitrum_one(
+        arbitrum_one_inquirer: 'ArbitrumOneInquirer',
+        arbitrum_one_transaction_decoder: 'ArbitrumOneTransactionDecoder',
+        arbitrum_one_accounts: list[ChecksumEvmAddress],
+        arbitrum_one_manager_connect_at_start,
+        inquirer: 'Inquirer',  # pylint: disable=unused-argument
+) -> None:
+    """Check that balances of GRT currently delegated to indexers are properly detected."""
+    wait_until_all_nodes_connected(
+        connect_at_start=arbitrum_one_manager_connect_at_start,
+        evm_inquirer=arbitrum_one_inquirer,
+    )
+    tx_hex = deserialize_evm_tx_hash('0x3c846f305330969fb0ddb87c5ae411b4e9692f451a7ff3237b6f71020030c7d1')  # noqa: E501
+    get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=arbitrum_one_transaction_decoder.database,
+        tx_hash=tx_hex,
+    )
+    thegraph_balances_inquirer = ThegraphBalancesArbitrumOne(
+        database=arbitrum_one_transaction_decoder.database,
+        evm_inquirer=arbitrum_one_inquirer,
+    )
+    thegraph_balances = thegraph_balances_inquirer.query_balances()
+    user_balance = thegraph_balances[arbitrum_one_accounts[0]]
+    assert user_balance.assets[A_GRT_ARB.resolve_to_evm_token()] == Balance(
+        amount=FVal('25.607552758075613'),
+        usd_value=FVal('38.4113291371134195'),
```

Comment:
the mocked price is 1.5. Can you set amount as a variable and then usd_value as `amount * FVal(1.5)`